core/src/main/resources/messages/messages.properties

```diff
@@ -101,6 +101,7 @@ error.no.device.request.present=No new device request present for mobile number
 error.device.request.expired=New device request expired for mobile number ''{0}''
 error.device.request.invalid.password=Invalid temporary password to use new device request for mobile number ''{0}''. Maybe the request was replaced with a different secret?
 error.device.unknown.operating.system=Operating system ''{0}'' is not supported.
+error.device.invalid.device.name=Device name ``{0}`` is invalid. A valid device name is not longer than {1} characters and does not contain a ''{2}''.
```

Comment:
The quotes are wrong here too.

Reply:
There were a few more of these. Corrected them all.